Frontend-v1-Original/components/layout/layout.tsx

```diff
@@ -45,7 +70,28 @@ export default function Layout({
           </>
         )}
         <SnackbarController />
-        <main>{children}</main>
+        {!chain?.unsupported ? (
+          <main>{children}</main>
+        ) : (
+          <div className="flex flex-grow items-center justify-center text-center">
+            <div>
+              <Typography className="max-w-md text-2xl text-white">
+                The chain you&apos;re connected to isn&apos;t supported. Please
+                check that your wallet is connected to Canto Mainnet.
```

Comment:
I found those escaped entities hard to maintain, why not use `{}` here, e.g.,

```suggestion
               {`The chain you're connected to isn't supported. Please
                check that your wallet is connected to Canto Mainnet.`}
```

Reply:
Hm. Could you elaborate why it's hard to maintain?

Reply:
Don't u think they're harder to read, and difficult to input? [The eslint rule](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md) exists for some reasons,

> This rule prevents characters that you may have meant as JSX escape characters from being accidentally injected as a text node in JSX statements.

but I don't think it asks us to use the escaped entities whenever possible.

To me I might rather disable the rule or use the `{``}` version.